cli/src/main/java/com/box/l10n/mojito/cli/command/PullCommand.java

```diff
@@ -87,6 +87,9 @@
     @Parameter(names = {"--async-ws"}, required = false, description = "Use async WS, use for processing big files (should become default eventually)")
     Boolean asyncWS = false;
 
+    @Parameter(names = {"--untranslated-format"}, required = false, hidden = true, description = "Format for untranslated strings. The expected format for the pattern is: \"prefix{source}suffix\". In order to have the source string displayed, the string: {source} needs to be present in the pattern.")
```

Comment:
`--use-parent-untranslated-format` ? or you kept it short on purpose?

Reply:
Naming fix: https://github.com/box/mojito/pull/772